jans-linux-setup/jans_setup/setup_app/installers/jans_lock.py

```diff
@@ -10,6 +10,8 @@
 from setup_app.installers.jetty import JettyInstaller
 
 Config.jans_lock_port = '8076'
+Config.jans_opa_host = 'localhost'
+Config.jans_opa_port = '8081'
```

Comment:
Config.jans_opa_port shouldn't be 8081 as it's used by `jans-auth`, instead `8181` should be used as seen in previous commit.

Reply:
Done